src/styles/main.scss

```diff
@@ -297,6 +297,12 @@ ol {
 }
 
 
+svg {
```

Comment:
So, this isn't working exactly as you think. In order to use SVGs for a `clip-path`, you need to refer to the ID of an element inside of the current document; it doesn't work with externally-referenced files (including data URIs). So there's a [root-level SVG](https://github.com/mozilla/universal-search-content/blob/master/src/templates/icons.html) that hangs out inside of `.index` (should probably be outside, but there's an Ampersand restriction that you can only have one root element for a view) that the CSS references like `clip-path: url(#bookmark)`.

I moved that out to a subview that just rendered a template to nicely separate concerns, but that's just needless overhead. I'll move that back out to the template.

Since that `<svg>` element hangs out inside of `.index`, it factors into the height calculation, hence this code. Point absolutely taken about using a class though; I generally default to using the least specific selector possible, but it makes a lot of sense to use a class for something so sweeping.